subsys/bluetooth/audio/Kconfig.bap

```diff
@@ -211,6 +211,7 @@ config BT_BAP_SCAN_DELEGATOR
 	select BT_EXT_ADV
 	select BT_PER_ADV_SYNC
 	select BT_ISO_SYNC_RECEIVER
+	select BT_GATT_DYNAMIC_DB
```

Comment:
```suggestion
	depends on BT_GATT_DYNAMIC_DB
```

Reply:
> Then I have to add it to all related configurations...

Yep :D 

But really we should always try to use `depends on` rather than `select` for various reasons: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#alternatives-to-select

Reply:
OK.